src/librustc_mir/build/scope.rs

```diff
@@ -107,6 +107,9 @@ pub struct Scope<'tcx> {
     /// `ScopeAuxiliary`, but kept here for convenience
     extent: CodeExtent,
 
+    /// Whether there's anything for the cleanup path.
```

Comment:
We should probably *always* refer to that as unwind, not cleanup, the latter is confusing, but I took it from the context. This field is true iff there's *anything* to be done for this scope when unwinding.